samples/server/petstore/php-symfony/SymfonyBundle-php/Api/PetApiInterface.php

```diff
@@ -94,7 +94,7 @@ public function addPet(Pet $pet, &$responseCode, array &$responseHeaders);
      * @param  integer $responseCode     The HTTP response code to return
      * @param  array   $responseHeaders  Additional HTTP headers to return with the response ()
      *
-     * @return void
+     * @return \void
```

Comment:
Is this also necessary for scalar types? Construction `@return \void` looks weird. Anyway it's just a comment, it cannot break anything, I'm just curious. Don't even need to test it since it's doc block change.

Reply:
It does not really matter perse